.editorconfig

```diff
@@ -0,0 +1,11 @@
+# EditorConfig: https://EditorConfig.org
+# Notepad++ Plugin: https://github.com/editorconfig/editorconfig-notepad-plus-plus
+# Visual Studio Code Plugin: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
+
+root = true
+
+[*.{css,html,js,json,scss,xml}]
```

Comment:
There is no xml file in this project, but there are *.svg :)

Reply:
LOL, hawk eyes. I think found a perfectionist with sharper eyes than me. 😄